optimum/neuron/pipelines/transformers/base.py

```diff
@@ -171,7 +171,7 @@ def load_pipeline(
             model, token=token, revision=revision
         ):
             logger.info("Using Sentence Transformers compatible Feature extraction pipeline")
-            neuronx_model_class = NeuronModelForSentenceTransformers
+            neuronx_model_class = NeuronSentenceTransformers
```

Comment:
nit: what about calling this class `NeuronSentenceTransformersModel` to be aligned with the naming.